zanata-war/src/main/java/org/zanata/service/impl/DocumentServiceImpl.java

```diff
@@ -266,12 +271,19 @@ private void processWebHookDocumentMilestoneEvent(
 
                 if (shouldPublish) {
                     HDocument document = documentDAO.getById(event.getDocumentId());
+
+                    String docUrl =
+                            urlUtil.fullEditorDocumentUrl(project.getSlug(),
+                                version.getSlug(), event.getLocaleId(),
+                                LocaleId.EN_US, document.getDocId());
+
                     DocumentMilestoneEvent milestoneEvent =
                             new DocumentMilestoneEvent(project.getSlug(),
                                     version.getSlug(), document.getDocId(),
-                                    event.getLocaleId(), message);
+                                    event.getLocaleId(), docUrl, message);
```

Comment:
Perhaps the URL should have been added at the end of the field list in DocumentMilestoneEvent, (and thus the end of the constructor params).